templates/static/team.html

```diff
@@ -6,6 +6,14 @@
 
 {% block description %}{% trans "Sefaria's team is working to create the Jewish future." %}{% endblock %}
 
+{% block css %}
+    #staticContentWrapper #content {
+      display: flex;
+      flex-direction: row;
+    }
+
+{% endblock %}
```

Comment:
See if you can use a subclass of interface-{lang} to get it to work instead of clock css

would end up something like:

style.css:
```css
      .interface-he {
        direction: rtl;
        text-align: right;
      }
      .interface-he .flex {
        display: flex;
        flex-direction: row-reverse;
      }

``` 

base.html
```html
    {% if not html %}
    <div id="staticContentWrapper">
        <div id="content" class="interface-he">
            <div class="flex"></div>
        </div>
    </div>
```

Reply:
setting proper direction here should also let you not bother with setting flex direction separately fro hebrew interface?